### PR TITLE
Bug 1182630 - Properly highlight the advanced call controls when disabled r=drs

### PR DESCRIPTION
**Pull request by gabrielesvelto**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=1b130591a9eebd6cae59f80c74aa21833f93dbc3

File: apps/callscreen/style/oncall.css [hunk 1]
```diff
@@ -284,24 +284,27 @@
...
-  .co-advanced-option:active .icon.icon-bt {
-    background-position: 0 0;
+  /* When active show the light blue icon */
+  .co-advanced-option.active-state .icon.icon-on-hold {
+    background-position: 0 8rem;
   }
 
-  .co-advanced-option.active-state:active .icon.icon-on-hold {
+  /* When pressed show the grey icon if the control is enabled */
+  .co-advanced-option.active-state:not([disabled]):active .icon.icon-on-hold {
```
Comment (DouglasSherk) [inline apps/callscreen/style/oncall.css]: The comment here is inaccurate.

File: apps/callscreen/style/oncall.css [hunk 2]
```diff
@@ -284,24 +284,27 @@
...
-  .co-advanced-option:active .icon.icon-speaker,
-  .co-advanced-option:active .icon.icon-bt {
-    background-position: 0 0;
+  /* When active show the light blue icon */
+  .co-advanced-option.active-state .icon.icon-on-hold {
+    background-position: 0 8rem;
   }
 
-  .co-advanced-option.active-state:active .icon.icon-on-hold {
+  /* When pressed show the grey icon if the control is enabled */
```
Comment (DouglasSherk) [inline apps/callscreen/style/oncall.css]: I think that this comment, and the one at line 301, are reversed.

Comment (gabrielesvelto) [inline apps/callscreen/style/oncall.css]: Yeah, actually this is quite strange. The light blue active icon is in the middle of the image (4rem) while the grey active icon is at the bottom (8rem) yet these selectors give the right result. I've even checked with the inspector and it works correctly: 8rem gives me the light blue icon and 4rem gives me the grey one. I have no idea what's going on but in principle it shouldn't work yet it does. I'll have a look next Monday before landing.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=bddde851258bd25cd15f85302ab1e65d4d7e0884

**Pull request merged**